src/components/WeddingCard/WeddingCard.tsx

```diff
@@ -1,25 +1,27 @@
 import './WeddingCard.css';
 import React from 'react';
+import { Link } from 'react-router-dom'
 
 type Wedding = {
-  key: number;
+  weddingId: number;
   name: string;
   image: string;
   date: string;
 }
 
 const WeddingCard: React.FC<Wedding> = ({
+  weddingId,
   name,
   image,
   date}) => {
     return (
-        <article className="cardWrapper">
+        <Link to={`/wedding/${weddingId}`} className="cardWrapper">
```

Comment:
Good call on using Link to change the route! We'll need to make sure to get rid of the text decoration in the styling for this now though.